ui/redux/actions/publish.js

```diff
@@ -598,6 +599,14 @@ export const doPrepareEdit = (claim: StreamClaim, uri: string, fileInfo: FileLis
   }
 
   dispatch({ type: ACTIONS.DO_PREPARE_EDIT, data: publishData });
+  /* Temporary solution for redirects depending on the publish type */
+  if (stream_type === 'document') {
+    dispatch(push(`/$/${PAGES.POST}`));
+  } else if (stream_type === undefined) {
+    dispatch(push(`/$/${PAGES.LIVESTREAM}`));
+  } else {
+    dispatch(push(`/$/${PAGES.UPLOAD}`));
+  }
```

Comment:
For PDFs, `stream_type = document`, but the action should be considered as a file-upload, not post-edit.

Reply:
Fixed.